src/main/resources/application.properties

```diff
@@ -34,4 +34,4 @@ spring.mail.properties.mail.smtp.auth=true
 spring.mail.properties.mail.smtp.starttls.enable=true
 mail.debug=false
 
-hostname=localhost:3000
```

Comment:
maybe `host.name`  and `host.port` ?

Reply:
also I hope `google.api.key` and aws secret are not real

Reply:
No need of host.port
Just need to put host.name=localhost:3000
Not sure about google and aws secrets looks fine to me

Thanks